matchmaking-ext/end_participation.go

```diff
@@ -11,7 +11,35 @@ import (
 
 func endParticipation(err error, client *nex.Client, callID uint32, idGathering uint32, strMessage string) {
 	server := commonMatchMakingExtProtocol.server
-	matchmakeSession := common_globals.Sessions[idGathering].GameMatchmakeSession
+	session, ok := common_globals.Sessions[idGathering]
+	if(!ok){
```

Comment:
```suggestion
	if !ok {
```

Reply:
Same comment as before for me. The more idiomatic way of writing these statments in Go is

```suggestion
	if session, ok := common_globals.Sessions[idGathering]; !ok {
```